tests/test_configuration.py

```diff
@@ -47,3 +46,17 @@ def test_argument_parsing_empty_toml(self, mock_sys):
         self.assertListEqual(
             configuration.scan_arguments, ["-DSomeJVMArg", "-DAnotherJVMArg", "-dNotAJVMArg", "-SomeNonsense"]
         )
+
+        mock_sys.argv = ["path/to/scanner/py-sonar-scanner", "-Dtoml.path=tests/pyproject.toml"]
+        configuration.setup()
+        self.assertListEqual(
+            configuration.scan_arguments, ["-Dtoml.path=tests/pyproject.toml", "-Dsonar.a=b", "-Dsonar.c=d"]
+        )
+
+        mock_sys.argv = ["path/to/scanner/py-sonar-scanner", "-Dproject.home=tests"]
+        configuration.setup()
+        self.assertListEqual(configuration.scan_arguments, ["-Dproject.home=tests", "-Dsonar.a=b", "-Dsonar.c=d"])
+
+        mock_sys.argv = ["path/to/scanner/py-sonar-scanner", "-Dproject.home=tests2"]
+        configuration.setup()
+        self.assertListEqual(configuration.scan_arguments, ["-Dproject.home=tests2"])
```

Comment:
So I noticed that we split the argument around the `=` token. Would it be possible to add a test that shows that if an argument has multiple `=` tokens, then we only take the first two elements? I think a test to showcase that this is the intended use might be useful. To make it clear here is what I mean:
```java
mock_sys.argv = ["path/to/scanner/py-sonar-scanner", "-Dproject.home=tests1=tests2"]
configuration.setup()
self.assertListEqual(configuration.scan_arguments, ["-Dproject.home=tests1"])
```